src/libc.c

```diff
@@ -4,6 +4,7 @@
 #include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <sys/mount.h>
```

Comment:
nice job - sorry about that. I should have checked https://man.freebsd.org/cgi/man.cgi?query=statfs.

I think we also need add `#include <sys/statvfs.h>` for `ST_LOCAL`, judging from https://man.netbsd.org/statvfs.5

Reply:
Good idea.  I'll try to test that in my NetBSD VM.

Reply:
NetBSD actually does `#define ST_LOCAL MNT_LOCAL`
I didn't test it but I assume it works